### ISSUE TITLE
Flexor hallucis longus tendon injury

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11178]
**Preferred term label:**
Flexor hallucis longus tendon injury

**Synonyms**
FHL

**Definition (free text, please give PubMed ID)**
MRI or ultrasound of the ankle/foot showing degenerative changes of the FHL tendon such as fluid around the tendon at the level of the ankle joint, intra-substance tendinous signal, tendon thickening, tendonitis, or tenosynovitis, or tendon rupture

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Flexor hallicus longus (FHL) tendon dysfunction
Os trigonum syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001400
- label: Flexor hallucis longus tendon injury
- parents: [Abnormal hallux morphology (HP:0001844)](https://hpo.jax.org/app/browse/term/HP:0001844)
- pmid: 
- def: Damage to the flexor hallucis longus (FHL) tendon. The FHL tendon is a long tendon that runs from the deep calf muscle down through the ankle to the big toe, enabling toe flexion and supporting foot mechanics.. MRI or ultrasound of the ankle/foot shows degenerative changes of the FHL tendon such as fluid around the tendon at the level of the ankle joint, intra-substance tendinous signal, tendon thickening, tendonitis, or tenosynovitis, or tendon rupture.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11178
- synonyms:
